Modules/_io/_iomodule.h

```diff
@@ -140,7 +144,7 @@ extern Py_off_t PyNumber_AsOff_t(PyObject *item, PyObject *err);
 
 extern PyModuleDef _PyIO_Module;
 
-typedef struct {
+typedef struct _io_state {
```

Comment:
Please remove the typedef here to prevent compiler warnings/errors. clang dislikes this if I recall correctly.

Reply:
Removed in 7aa8da76a45c237f80e86a4bd2d0542e8426e395